vllm/model_executor/models/siglip.py

```diff
@@ -344,10 +344,16 @@ def __init__(
 
         self.config = config
         self.activation_fn = get_act_fn(config.hidden_act)
-
-        # For quantization, we require the hidden size to be a multiple of 64
-        quantizable = (config.hidden_size % 64 == 0
-                       and config.intermediate_size % 64 == 0)
+        # Special handling for BNB quantization
+        if quant_config.__class__.__name__ == "BitsAndBytesConfig":
```

Comment:
```suggestion
        if quant_config and quant_config.get_name() == "bitsandbytes":
```

I think we can use the `get_name` method from `quant_config` here.

Reply:
Done